trustpinning/trustpin.go

```diff
@@ -121,3 +126,23 @@ func getPinnedCAFilepathByPrefix(gun data.GUN, t TrustPinConfig) (string, error)
 	}
 	return specificCAFilepath, nil
 }
+
+// wildcardMatch will attempt to match the most specific (longest prefix) wildcarded
+// trustpinning option for key IDs. Given the simple globbing and the use of maps,
+// it is impossible to have two different prefixes of equal length.
+// This logic also solves the issue of Go's randomization of map iteration.
+func wildcardMatch(gun data.GUN, certs map[string][]string) ([]string, bool) {
+	var (
+		longest = ""
+		ids     []string
+	)
+	for k, v := range certs {
```

Comment:
can we rename `k` and `v` to something more descriptive like `gunPrefix` and `keyIDs`?